src/telemetry/Counter.cc

```diff
@@ -0,0 +1,53 @@
+#include "zeek/telemetry/Counter.h"
+
+#include "zeek/telemetry/Manager.h"
+
+#include "opentelemetry/metrics/provider.h"
+
+using namespace zeek::telemetry;
+
+IntCounterFamily::IntCounterFamily(std::string_view prefix, std::string_view name, Span<const std::string_view> labels,
+                                   std::string_view helptext, std::string_view unit, bool is_sum,
+                                   opentelemetry::metrics::ObservableCallbackPtr callback)
```

Comment:
That function takes a `nullptr` by default so it should be invisible to other callers. That said, I haven't taken a whole lot of care with this to avoid API changes.